static/index.js

```diff
@@ -52,4 +61,19 @@ document.addEventListener("DOMContentLoaded", event => {
     // right mouse button click
     ws.send(JSON.stringify({ type: "doubletap" }));
   });
+  doubleTap.recognizeWith(singleTap);
+  doubleTap.requireFailure(singleTap);
+  singleTap.requireFailure(doubleTap);
+
+  hscrollHammer.add(scrollx);
+  vscrollHammer.add(scrolly);
+
+  hscrollHammer.on("scrollx", ev => {
+    // scroll
+    ws.send(JSON.stringify({ type: "scroll" }));
```

Comment:
Is it intended to same the same event for horizontal and vertical scroll? Looks like we are sending the same event and only applying `scroll` (which is vertical scroll) on the server side.

Reply:
Hi! No, it was not intended. I have fixed this issue in my latest push!